tooling-tests/src/test/resources/test-src/generated-sources/tool_test_unary_11/emptyFieldMessage_pb_client.bal

```diff
@@ -0,0 +1,87 @@
+import ballerina/grpc;
+import ballerina/protobuf;
+import ballerina/protobuf.types.empty;
+
+public const string EMPTYFIELDMESSAGE_DESC = "0A17656D7074794669656C644D6573736167652E70726F746F1204746573741A1B676F6F676C652F70726F746F6275662F656D7074792E70726F746F22410A0B436F6D706C65785479706512100A03666F6F18012001280D5203666F6F12200A0362617218022001280E320E2E746573742E54657374456E756D520362617222790A134F7074696F6E616C436F6D706C65785479706512360A0D636F6D706C65785F76616C756518012001280B32112E746573742E436F6D706C657854797065520C636F6D706C657856616C7565122A0A046E6F6E6518022001280B32162E676F6F676C652E70726F746F6275662E456D70747952046E6F6E6522500A0E4F7074696F6E616C537472696E6712120A046E616D6518012001280952046E616D65122A0A046E6F6E6518022001280B32162E676F6F676C652E70726F746F6275662E456D70747952046E6F6E65223D0A0E526571756573744D657373616765122B0A0372657118012003280B32192E746573742E4F7074696F6E616C436F6D706C6578547970655203726571223B0A0F526573706F6E73654D65737361676512280A047265737018012003280B32142E746573742E4F7074696F6E616C537472696E675204726573702A280A0854657374456E756D120D0A09454E5452595F4F4E451000120D0A09454E5452595F54574F100132450A0B546573745365727669636512360A075465737452504312142E746573742E526571756573744D6573736167651A152E746573742E526573706F6E73654D657373616765620670726F746F33";
```

Comment:
ditto

Reply:
This is a generated file. Yes, we will improve the code with a separate task.